inst/tests/tests.Rraw

```diff
@@ -17186,3 +17186,17 @@ DT = data.table(value=as.list(1:6), index=rep(1:2, each=3))
 test(2158.2, DT[, by="index", list(value=list(value))],
              data.table(index=1:2, value=list(as.list(1:3), as.list(4:6))))
 
+# print shows the total number of rows when exceeding the max.print lines PR#4778
+old = options(max.print = 5L)
+DT = as.data.table(iris)
+test(2159.1, print(DT), output="150 rows in total")
+options(old)
+# ensure invalid multi-byte strings can be printed correctly
+latin1 = "fa\xE7ile"
+Encoding(latin1) = "latin1"
+utf8 = iconv(latin1, "latin1", "UTF-8")
+text <- c(latin1, utf8, "aaaaaaaa")
+Encoding(text)[1] = c('UTF-8')
+test(2159.2, print(data.table(A = text)), output=c(
+  "           A", "1: fa\\xe7ile", "2:    façile", "3:  aaaaaaaa")
```

Comment:
Pesky non-ascii `ç` here. Just needs replacing with `\\xe7` like the others, iiuc.

Reply:
Done.